desktop/core/requirements.txt

```diff
@@ -9,7 +9,7 @@ channels-redis==2.3.2
 configobj==5.0.6
 cryptography==3.2
 django-auth-ldap==1.3.0
-Django==2.0.13 # Django 2 then 3?
+Django==2.1.15 # Django 2 then 3?
```

Comment:
Should we do this on in a separate commit?

Reply:
yes, just push for checking that it is working in CI or not.